agent/src/main/java/com/newrelic/agent/android/NewRelic.java

```diff
@@ -1061,8 +1094,12 @@ public static boolean recordJSErrorException(StackTrace stackTrace) {
         return DataController.sendAgentData(stackTrace);
     }
 
-
-    public static boolean  addHTTPHeadersTrackingFor(List<String> headers) {
+    /**
+     * Adds a set of request header instrumentation targets
```

Comment:
@ndesai-newrelic We could use a better description here. Could you please provide a better one?

Reply:
@cthomas-newrelic i will update it.